src/hyp3_autorift/process.py

```diff
@@ -520,6 +565,7 @@ def main():
     )
     parser.add_argument('--bucket', help='AWS bucket to upload product files to')
     parser.add_argument('--bucket-prefix', default='', help='AWS prefix (location in bucket) to add to product files')
+    parser.add_argument('--opendata-upload', type=bool, default=True, help="If or not upload to its-live-data bucket")
```

Comment:
I'd prefer something like `--publish`
```suggestion
    parser.add_argument('--publish', type=string_is_true, default=False, help=f'Additionally publish the product to the ITS_LIVE AWS Open Data bucket: s3://{OPEN_DATA_BUCKET}')
```
and we want the default to be `False` (only publish when explicitly told to!) and HyP3 has to pass a string to this option so the type has to be `string_is_true`, which is defined in `hyp3lib.util` , e.g.:
https://github.com/ASFHyP3/hyp3-gamma/blob/develop/hyp3_gamma/__main__.py#L91